helm/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. You can access immudb web console by running these commands:
+{{- if .Values.ingress.enabled }}
+  curl http{{ if $.Values.ingress.tls }}s{{ end }}://{{ .Values.ingress.hostname }}/
+  immuclient status -a {{ .Values.ingress.hostname }} -p 80 
```

Comment:
Shouldn't the port depend on the tls setting?

Reply:
not really... immuclient afaik doesn't support https, so you have to use 80

Reply:
Immuclient itself does support tls connection - even with mutual tls where the client authenticates to immudb using client cert. I'm not sure however how would it work with the ingress. Maybe just setting the port to 443 and adding the --certificate option with the CA generated by kubernetes (and I assume that this wouldn't be needed if there was letsencrypt cert in front for example) would work out-of-the-box?

Reply:
Mutual tls is complicated, as we need to get the client certificate on immudb. Possible, but I'd leave it out this initial release.
I wasn't able to find a way to use immuclient in the "traditional" https way (i.e., not mutual tls).
In any case, port 80 seems to keep working even with tls enabled, so I'll let it that way for the moment.

Reply:
I was thinking  that maybe we shouldn't expose immudb's protocol through ingress at all - it will rather be used internally by other pods (in which case there's always the service object), and we can expose the http interface for the web console. For native protocols such as the immudb grpc one, I think people should use rather nodeport or loadbalancer.